paper-menu-button.html

```diff
@@ -79,28 +79,31 @@
         @apply(--paper-menu-button-disabled);
       }
 
-      :host([vertical-align="top"]) paper-material {
-        margin-bottom: 20px;
-        margin-top: -10px;
-        top: 10px;
-      }
-
-      :host([vertical-align="bottom"]) paper-material {
-        bottom: 10px;
-        margin-bottom: -10px;
-        margin-top: 20px;
-      }
-
       iron-dropdown {
         @apply(--paper-menu-button-dropdown);
       }
 
       .dropdown-content {
+        @apply(--shadow-elevation-2dp);
```

Comment:
Since this is a class we required the user to use, which means they can style it in the light dom, do you think we should use a different class name or it doesn't matter?